MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -397,7 +411,15 @@ + (void)destroyPlayer:(NSString *)name {
 
 #pragma mark Video Change
 
-+ (void)videoChangeForPlayer:(nonnull NSString *)name  withPlayerData:(nullable MUXSDKCustomerPlayerData *)playerData withVideoData:(nullable MUXSDKCustomerVideoData *)videoData viewData: (nullable MUXSDKCustomerViewData *) viewData {
++ (void)videoChangeForPlayer:(nonnull NSString *)name withCustomerData:(nullable MUXSDKCustomerData *)customerData {
+    MUXSDKCustomerPlayerData *playerData = [customerData customerPlayerData];
+    if (!playerData) {
+        playerData = [_customerPlayerDataStore playerDataForPlayerName:name];
+    }
+    MUXSDKCustomerViewData *viewData = [customerData customerViewData];
+    MUXSDKCustomerVideoData *videoData = [customerData customerVideoData];
+    MUXSDKCustomData *customData = [customerData customData];
+    
     if (!(videoData || viewData)) {
```

Comment:
need to add `|| customData` here

Reply:
So...this is a draft PR lol (I'm still working on it)

But this comment just saved me! I was trying to figure out why one of my tests cases was not working for the last hour and this is the reason 🙏

Reply:
okay now this is ready